addons/mail/static/src/models/thread/thread.js

```diff
@@ -206,38 +206,34 @@ function factory(dependencies) {
             // relations
             if ('members' in data) {
                 if (!data.members) {
-                    data2.members = [['unlink-all']];
+                    data2.members = unlinkAll();
                 } else {
-                    data2.members = [
-                        ['insert-and-replace', data.members.map(memberData =>
-                            this.env.models['mail.partner'].convertData(memberData)
-                        )],
-                    ];
+                    data2.members = insertAndReplace(data.members.map(memberData =>
+                        this.env.models['mail.partner'].convertData(memberData)
+                    ));
```

Comment:
This is the one I was referring to earlier, it needs to stay a list of commands and not a single command.